classes.py

```diff
@@ -53,15 +53,15 @@ def turn_off(self, game_state):
         return game_state
 
 
-    def perform_card_effect(self, game_state, card_effect):
+    def perform_CardEffect(self, game_state, CardEffect):
```

Comment:
Tohle je pořád jméno metody, píše se s malýma písmenama.

Reply:
def perform_card_effect(self, game_state, CardEffect): ??? takto

Reply:
To `card_effect` je parametr funkce, takže `def perform_card_effect(self, game_state, card_effect):`

Reply:
@vahalova za me jsou zmeny v PR ok